sdk/core/test/helpers/getOtpFromTestMailApp.ts

```diff
@@ -0,0 +1,46 @@
+let fetch: any
+
+/* istanbul ignore next */
+if (!fetch) {
+  fetch = require('node-fetch')
+}
+
+export class EmailConfiguration {
+  public static readonly TESTMAIL_URL = 'https://api.testmail.app'
+  public static readonly TESTMAIL_API_KEY = process.env.TESTMAIL_API_KEY
+  public static readonly TESTMAIL_NAMESPACE = process.env.TESTMAIL_NS
+  public static readonly TESTMAIL_PATH = '/api/json'
+}
+
+export interface TestEmail {
+  subject: string
+  html: string
+  text: string
+  downloadUrl: string
+  from: string
+  to: string
+  tag: string
+  timestamp: number
+}
+
+export class TestEmailHelper {
+  public static generateEmailForTag = (tag: string): string =>
+    `${EmailConfiguration.TESTMAIL_NAMESPACE}.${tag}@inbox.testmail.app`
+  public static getEmailsForTag = async (tag: string): Promise<TestEmail[]> => {
+    const url = `${EmailConfiguration.TESTMAIL_URL}${EmailConfiguration.TESTMAIL_PATH}?apikey=${EmailConfiguration.TESTMAIL_API_KEY}&namespace=${EmailConfiguration.TESTMAIL_NAMESPACE}&tag=${tag}`
+    const request = await fetch(url)
+    const data = await request.json()
+    const emails = data.emails.map(
+      (m: any): TestEmail => {
+        const { from, to, subject, html, downloadUrl, tag, timestamp, text } = m
+        return { from, to, subject, html, downloadUrl, tag, timestamp, text }
+      },
+    )
+    return emails
+  }
+  public static extractOpt = async (tag: string) => {
+    const [{ text, html }] = await TestEmailHelper.getEmailsForTag(tag)
+    const matches = (html || text || '').match(/\d{6}/)
```

Comment:
Not sure that's a good idea to extract codes like that. And it is not compatible with tests in this PR:

```
>     const stamp = `${Date.now()} stamp`
undefined
>     const delimiter = ':'
undefined
>     const messageParameters = {
...       message: `Your verification code is ${delimiter}${stamp}${delimiter}{{CODE}}.`,
...       subject: `${stamp}${delimiter}{{CODE}}`,
...     }
undefined
> const text = messageParameters.message.replace('{{CODE}}', '123456')
undefined
> const matches = text.match(/\d{6}/)
undefined
> matches[matches.length-1]
'161968'
```

Reply:
We need to check not just code but the somehow the custom message that we pass as a template. so that I try to bake some indicator to a string with delimiters

Reply:
Not sure how is this comment related to `extractOtp` method?